src/main.rs

```diff
@@ -19,6 +19,19 @@ mod error;
 mod matching;
 mod parsing;
 mod pe;
+#[cfg(not(llvm_13))]
+fn get_pdb(pe: BaoPE) {
+    return pdb_wrapper::PDB::new(pe.is_64)?;
+}
+
+#[cfg(llvm_13)]
+fn get_pdb(pe: BaoPE) {
```

Comment:
This shouldn't compile. There's no return type and not all paths return a value

Reply:
added return type in later commit